person.rb

```diff
@@ -28,5 +28,5 @@ def add_rental(person, date)
     Rental.new(date, self, person)
```

Comment:
The user is not able to create a rental. Kindly see the error which occurs while creating the label.

![rental](https://user-images.githubusercontent.com/35504149/189356725-b944b861-34eb-406a-a36f-0c4ed3ca4983.png)

Reply:
Thanks i have solved it